six/assetlist.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "six",
+    "assets": [
+      {
+        "description": "The native staking token of the SIX Protocol.",
+        "denom_units": [
+          {
+            "denom": "usix",
+            "exponent": 0
+          },
+          {
+            "denom": "six",
+            "exponent": 6
+          }
+        ],
+        "base": "usix",
+        "name": "Six token",
+        "display": "six",
+        "symbol": "SIX",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/six/images/six.png"
```

Comment:
we are OK to proceed, but note that due to the aspect ratio of the image not being exactly square, it could end up looking squished on some interfaces that stretch the image to force it to be square.

Reply:
Ok, that's acceptable for us. Thank you.